pyseir/inference/model_fitter.py

```diff
@@ -418,12 +459,28 @@ def _fit_seir(self, R0, t0, eps, t_break, test_fraction, hosp_fraction, log10_I_
         Returns
         -------
           : float
-            Chi square of fitting model to observed cases and deaths.
+            Chi square of fitting model to observed cases, deaths, and hospitalizations.
         """
         l = locals()
         model_kwargs = {k: l[k] for k in self.model_fit_keys}
-        model = self.run_model(**model_kwargs)
 
+        # Last data point used in Fit
+        last_data_point_used = t0 + t_break + 14 + t_delta_phases + 14
+        # Number of future days used in second ramp period
+        number_of_future_days_used = last_data_point_used - self.ref_future_date
+        # Multiplicative chi2 penalty if future_days are used in second ramp period (set to 1 by default)
+        future_days_penalty = 1.0
```

Comment:
This is very low. seems it could be at least 10x this to even be of order the size of the same size as the data chi2

Reply:
This is the default penalty. I only change it if future days are used